src/argilla/server/schemas/v1/datasets.py

```diff
@@ -55,9 +55,13 @@
 QUESTION_CREATE_DESCRIPTION_MAX_LENGTH = 1000
 
 RATING_OPTIONS_MIN_ITEMS = 2
-RATING_OPTIONS_MAX_ITEMS = 100
+RATING_OPTIONS_MAX_ITEMS = 10
 
-VALUE_TEXT_OPTION_VALUE_MIN_LENGHT = 1
+RATING_LOWER_VALUE_ALLOWED = 1
```

Comment:
```suggestion
RATING_LOWER_VALUE_ALLOWED = 1
```
Makes sense - allows easier shortcut.
Presumably 10 can be selected with 0?